.github/workflows/sprint-issues-rollover.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow will move all unfinish tickets to the next sprint
+
+name: "Sprint Issues Rollover"
+
+on:
+    schedule:
+        # Runs every Wednesday at 6pm EST
+        - cron: '0 22 * * 3'
```

Comment:
We can also make it a manual trigger incase some our our weekly meetings get rescheduled